<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductInfo</class>
 <widget class="QWidget" name="ProductInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_productName">
      <property name="text">
       <string>&lt;Product_name&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_productEdit">
      <property name="icon">
       <iconset resource="../rec.qrc">
        <normaloff>:/resources/settings_icon.png</normaloff>:/resources/settings_icon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>208</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_protein">
      <property name="text">
       <string>Белки:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_numProtein">
      <property name="text">
       <string>&lt;Num_protein&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_fats">
      <property name="text">
       <string>Жиры:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_numFats">
      <property name="text">
       <string>&lt;Num_fats&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_carbohydrates">
      <property name="text">
       <string>Углеводы:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_numCarbohydrates">
      <property name="text">
       <string>&lt;Num_сarbohydrates&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_kcal">
      <property name="text">
       <string>Ккал:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_numKcal">
      <property name="text">
       <string>&lt;Num_kcal&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_productDescription">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../rec.qrc"/>
 </resources>
 <connections/>
</ui>
